controllers/admin/AdminAddressesController.php

```diff
@@ -517,6 +517,11 @@ public function ajaxProcess()
                 echo json_encode(array('infos' => pSQL($customer['firstname']) . '_' . pSQL($customer['lastname']) . '_' . pSQL($customer['company'])));
             }
         }
+
+        if (Tools::isSubmit('contains_dni')) {
```

Comment:
based on what is done on `states`

Reply:
It should never happened. It's the legacy logic.. Don't ask me why it was like that :/

Reply:
FYI, it's separated behaviour, it's because of ajaxProcess, which process all ajax requests.
I change only this part to limit changes because it's for a patch version and the supplier page will be migrate in 1.7.7.